includes/class-wp-job-manager-recaptcha.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/**
+ * File containing the WP_Job_Manager_Recaptcha class.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Notices_Conditions_Checker class.
+ *
+ * @since 1.40.0
+ * @internal
+ */
+class WP_Job_Manager_Recaptcha {
+
+	/**
+	 * Stores static instance of class.
+	 *
+	 * @var WP_Job_Manager_Helper_Renewals The single instance of the class.
```

Comment:
As suggested by PSalm:

```suggestion
	 * @var WP_Job_Manager_Recaptcha The single instance of the class.
```

Reply:
Done in 7f4d9d052bc89ea9a2952811160ef6200bf0c535